pom.xml

```diff
@@ -210,7 +211,7 @@
          <executions>
            <execution>
              <id>check</id>
-             <phase>verify</phase>
```

Comment:
Re-enabled and fixed all failures relating to incompatible changes of public APIs